bolna/agent_manager/task_manager.py

```diff
@@ -1503,6 +1503,9 @@ async def process_audio_data_for_output(self, meta_info, message):
         if self.task_config["tools_config"]["output"]["format"] == "pcm" and meta_info.get('format', '') != 'mulaw':
             message['data'] = wav_bytes_to_pcm(message['data'])
 
+        if self.synthesizer_provider == 'elevenlabs' and self.tools["output"].get_provider() == 'plivo':
```

Comment:
@prateeksachan Use elif over here.
plivo+elevelabs+pcm -> wav_bytes_to_pcm(message['data'] will be executed twice which is causing error.

Reply:
Thanks @KaliCharan-V . Have committed https://github.com/bolna-ai/bolna/pull/12/commits/2ee4605315ab6b1cff0ef43bf9e429bc0c11f156 for the next release today.